spec/inject_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'array'
+
+describe Inject do
+	it 'inject method...' do
+		expect(subject.innjecct([1,2,3,4],0)).to eq 10
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.